test/e2e/manage-type-external.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+source "${SRC_DIR}/setup-shell.sh"
+
+set -x # echo so that users can understand what is happening
+set -e # exit on error
+
+EXT_CLUSTER_NAME=ext
+
+:
+: -------------------------------------------------------------------------
+: Apply a post-create-hook for testing
+:
+kubectl --context kind-kubeflex apply -f ${SRC_DIR}/pod-pch.yaml
```

Comment:
It is a bit jarring that the filename here is very different from the name of the PCH defined inside.

Reply:
ok, renamed to be consistent